doc/manuals/user/walkthrough_apiv1.md

```diff
@@ -1025,8 +1025,12 @@ Let's examine in detail the different elements included in the payload:
     as in the example above, makes a notification not to be sent
     if a previous notification was sent less than 5 seconds ago, no
     matter how many actual changes take place in that period. This is give the 
-   notification receptor a means to protect itself against context producers 
-   that update attribute values too frequently.
+    notification receptor a means to protect itself against context producers
```

Comment:
Suggestions on how to explain this cleaner are welcome (please provide suggested wording).